<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoxelizeWidget</class>
 <widget class="QWidget" name="VoxelizeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VoxelizeWidget</string>
  </property>
  <layout class="QVBoxLayout" name="vlayout_widget">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="TitleBar" name="titleBar" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lb_voxelSize">
     <property name="text">
      <string>Voxel Size</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_voxelSize">
     <item>
      <layout class="QHBoxLayout" name="hlayout_voxelSizeX">
       <item>
        <widget class="QLabel" name="lb_voxelSizeX">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_voxelSizeX">
         <property name="text">
          <string>0.4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlayout_voxelSizeY">
       <item>
        <widget class="QLabel" name="lb_voxelSizeY">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_voxelSizeY">
         <property name="text">
          <string>0.4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlayout_voxelSizeZ">
       <item>
        <widget class="QLabel" name="lb_voxelSizeZ">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_voxelSizeZ">
         <property name="text">
          <string>0.4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lb_resolution">
     <property name="text">
      <string>Resolution</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_resolution">
     <item>
      <layout class="QHBoxLayout" name="hlayout_resolutionX">
       <item>
        <widget class="QLabel" name="lb_resolutionX">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_resolutionX">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlayout_resolutionY">
       <item>
        <widget class="QLabel" name="lb_resolutionY">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_resolutionY">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlayout_resolutionZ">
       <item>
        <widget class="QLabel" name="lb_resolutionZ">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_resolutionZ">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vlayout_solidModel">
     <item>
      <widget class="QLabel" name="lb_solidModel">
       <property name="text">
        <string>infill</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlayout_solidModel">
       <item>
        <widget class="QLabel" name="lb_solidModelSpace">
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ckb_solidModel">
         <property name="text">
          <string>Enable</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spr_widget">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>39</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btmbox_widget">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>lb_voxelSize</zorder>
  <zorder>lb_resolution</zorder>
  <zorder>spr_widget</zorder>
  <zorder>btmbox_widget</zorder>
  <zorder>titleBar</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header>titlebar.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
